[DOC-6100] Document that private clusters are required for CMEK (#15568)
cockroachdb/docs: cockroachcloud/managing-cmek.md

- [Customer-Managed Encryption Key (CMEK) frequently asked questions (FAQ)](cmek-faq.html)
- [Encryption at Rest (Enterprise)](../{{site.versions["stable"]}}/security-reference/encryption.html#encryption-at-rest)

## Prerequisites

> - You need a {{ site.data.products.db }} account with billing enabled. Egress Perimeter Controls are available only for {{ site.data.products.dedicated }}, not {{ site.data.products.serverless }} clusters, and {{ site.data.products.dedicated }} have a non-zero operating cost that must be billed.

Reviewer: Where does this point come from? This is not needed.

Author: I just misseed updating the feature name. I think this is true for CMEK as well, right?

> - You need a {{ site.data.products.db }} account with billing enabled. Egress Perimeter Controls are available only for {{ site.data.products.dedicated }}, not {{ site.data.products.serverless }} clusters, and {{ site.data.products.dedicated }} have a non-zero operating cost that must be billed.

Author: ```suggestion
- You need a {{ site.data.products.db }} account with billing enabled. CMEK is available only for {{ site.data.products.dedicated }}, not {{ site.data.products.serverless }} clusters, and {{ site.data.products.dedicated }} have a non-zero operating cost that must be billed.
```

- Your cluster must be a **Private Cluster**, with no public IP addresses on its nodes. Refer to [Private Clusters](private-clusters.html).

> - You need a service account with `admin` privilege on clusters in your organization. You can provision service accounts with API keys in the cloud console. Refer to [Service Accounts](console-access-management.html#service-accounts).

Author: ```suggestion
- You need a service account with `admin` privilege on clusters in your organization. You can provision service accounts with API keys in CockroachDB Cloud Console. Refer to [Service Accounts](console-access-management.html#service-accounts).
```

> - You need an API key with `admin` privilege on clusters in your organization. You can provision API keys in the cloud console: [Create API Keys](console-access-management.html#create-api-keys)

Reviewer: ```suggestion
- You need a service account with `admin` privilege on clusters in your organization. You can provision service accounts with API keys in the cloud console: [Service Accounts](console-access-management.html#service-accounts)
```
